### PR TITLE
Readme.md and few changes.

### PR DESCRIPTION
**Pull request by abt2804**
- New Readme.md
- New makefile
- Changed gloomwin.cpp to render 1024x768 because the window was too small.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Simply go into Gloom3D directory and type ``make``.
+
+
+Copyright
+------------
+
+Original 2000 version by Sasha Chukov, enhanced and updated for Haiku in 2018 by Arnav Bhatt.
+
+
+Contact
```
Comment (scottmc) [inline README.md]: Same here, you can drop the contact info.

Comment (arnavbhatt288) [reply README.md]: Done.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+Building
+--------
+
+Simply go into Gloom3D directory and type ``make``.
+
+
+Copyright
+------------
+
+Original 2000 version by Sasha Chukov, enhanced and updated for Haiku in 2018 by Arnav Bhatt.
```
Comment (scottmc) [inline README.md]: Same here, you can leave off the enhanced by comment.

**Comment by arnavbhatt288** [comment]
Sir, please merge this PR so that I can create a recipe for it. Thank you.

**Comment by scottmc** [comment]
Thanks.

**Pull request merged**